lib/robots/dor_repo/release/release_members.rb

```diff
@@ -17,16 +17,19 @@ def perform_work
           logger.debug "release-members working on #{druid}"
 
           return unless cocina_object.collection?
+          return unless add_wf_to_members?
 
-          publish_collection
+          members.each { |member| create_release_workflow(member) }
         end
 
         private
 
-        def publish_collection
-          member_service = Dor::Release::MemberService.new(druid:)
-          add_workflow_to_members(member_service.items) if add_wf_to_members?
-          add_workflow_to_members(member_service.sub_collections)
```

Comment:
This all becomes easier because removing code for releasing sub_collections. Collections can't have collections as members.